lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.6.0-beta.4",
+  "version": "0.6.0-beta.5",
```

Comment:
Very minor but seems like a big enough update to make this 0..6.1

Reply:
Our stable is 0.5.X and the current lean mode is 0.6.X and when it is generally available, it will be 0.6.0. All current iterations are beta.* per NPM conventions.